HoloLens2Example/README.md

```diff
@@ -9,18 +21,42 @@ Supported Unreal versions | Supported device
 :-----------------: | :----------------:
 Unreal Engine 4.24+ | HoloLens 2
 
-## How to build and deploy the app
+
+## Contents
+
+| File/folder | Description |
+|-------------|-------------|
+| `Config` | HoloLens and default configuration files. |
+| `Content` | Project files and assets. |
+| `Source` | Mixed Reality Toolkit for Unreal UX Tools plugin components. |
+| `.gitignore` | Define what to ignore at commit time. |
+| `HoloLens2Example.uproject` | Unreal Engine project file. |
+| `README.md` | This README file. |
+
+## Prerequisites
+
+<!-- Need these -->
+
+# Setup
 
 Make sure you have Unreal Engine 4.24 or later installed on your computer. 
 * If you do not have a HoloLens 2 device, first follow the [App Packaging Instructions](https://docs.unrealengine.com/en-US/Platforms/AR/HoloLens2/HowTo/PackageApp/index.html) and then follow the [Emulator Quickstart Instructions](https://docs.unrealengine.com/en-US/Platforms/AR/HoloLens2/QuickStartEmulator/index.html) to deploy the app to the [HoloLens 2 Emulator](https://docs.microsoft.com/en-us/windows/mixed-reality/using-the-hololens-emulator)
 * If you have a HoloLens 2 device and want to run the app on device, first follow the [App Packaging Instructions](https://docs.unrealengine.com/en-US/Platforms/AR/HoloLens2/HowTo/PackageApp/index.html) and then follow the [Device Quickstart Instructions](https://docs.unrealengine.com/en-US/Platforms/AR/HoloLens2/QuickStartDevice/index.html) to deploy the app to your HoloLens 2
 * If you have a HoloLens 2 device and want to stream the app to device, follow the [Streaming Quickstart Instructions](https://docs.unrealengine.com/en-US/Platforms/AR/HoloLens2/QuickStartStreaming/index.html) to stream the app from the Unreal editor to your HoloLens 2
 
-## How to use the app
+## Running the sample
```

Comment:
I actually think "How to use the app" is a better descriptor of the below section, as it's instructing readers how to interact with the app after it's deployed and running on device or emulator. "Running the sample" would make more sense if the instructions were teaching the user how to get the app running in the first place. I don't think there's really a difference between this and "Setup".

Reply:
Agreed, but these sections are how the samples directory wants the README's structured: https://review.docs.microsoft.com/en-us/help/contribute/samples/concepts/readme-template?branch=master